pkgs/jovian-hardware-survey/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib
+, runCommand
+, ruby
+, steamdeck-firmware
+, dmidecode
+}:
+
+runCommand "jovian-hardware-survey" { } ''
+  mkdir -p $out/bin
+  (
+  echo "#!${ruby}/bin/ruby"
+  echo 'ENV["PATH"] = "${lib.makeBinPath [ steamdeck-firmware dmidecode ]}:#{ENV["PATH"]}"'
+  echo 'D20BOOTLOADER = "${steamdeck-firmware}/libexec/d20bootloader"'
+  cat ${./jovian-hardware-survey.rb}
```

Comment:
Who needs wrappers?

(Using `builtins.toJSON` would be even better, now that I think about it... But given the properties of Nix Paths, it won't be an issue.)